src/StepFunctionAuthoring/StepFunctionAuthoring.tsx

```diff
@@ -16,7 +16,7 @@ import {
 import {
     StephFunctionAuthoringFormState,
```

Comment:
Could you please fix this one too?  Otherwise looks good to me.

Reply:
Done. Thanks!